recipes/chimes-calculator/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "chimes-calculator" %}
+{% set version = "1.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rk-lindsey/chimes_calculator/archive/refs/tags/v1.0.0.zip
+  sha256: 367e84bcca6cf9901198d031fef3ccaea0e9fdd1600d58566c73fc9ec729dcae
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - ninja
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libchimescalc.so  # [linux]
+    - test -f $PREFIX/lib/libchimescalc_fortran.so  # [linux]
+    - test -f $PREFIX/lib/libchimescalc.dylib  # [osx]
+    - test -f $PREFIX/lib/libchimescalc_fortran.dylib  # [osx]
```

You can use this instead of having one for Linux and another one for macOS.

```suggestion
    - test -f $PREFIX/lib/libchimescalc_fortran${SHLIB_EXT}
```